.gitignore

```diff
@@ -1 +1,2 @@
 /target
+examples/bad-case/simple/select.output
```

Comment:
Not a good practice to ignore `.output`. Maybe add a `rm` in makefile?

Reply:
Not for regular case, but in this case, this example will always fail, if we delete this output file, then how will user `diff *result *output`?

Reply:
Got it. How about adding a comment here (in `.gitignore`, I'm not sure if there is a better place to specify this rule)

Reply:
Added

Reply:
👍